install-worker.sh

```diff
@@ -80,8 +80,17 @@ sudo mkdir -p /etc/kubernetes/manifests
 sudo mkdir -p /var/lib/kubernetes
 sudo mkdir -p /var/lib/kubelet
 sudo mkdir -p /opt/cni/bin
+sudo mkdir -p /etc/cni/net.d
 
 CNI_VERSION=${CNI_VERSION:-"v0.6.0"}
+# Add the CNI loopback device
```

Comment:
Its unclear how this change is addressing the issue. 
Could you please provide additional details?

Reply:
This fixed this issue on kubelet startup:
```
    dec 21 10:21:07 ip-10-0-1-189.eu-west-1.compute.internal kubelet[12670]: W1221 10:21:07.776122   12670 cni.go:172] Unable to update cni config: No networks found in /etc/cni/net.d
    dec 21 10:21:07 ip-10-0-1-189.eu-west-1.compute.internal kubelet[12670]: W1221 10:21:07.786670   12670 cni.go:172] Unable to update cni config: No networks found in /etc/cni/net.d
```
Those error lines disappeared.

I do not know if the loopback CNI device is necessary, I copied it from @albertrdixon changes to his fork. I went through his code and pulled changes back into our fork.